packages/table/preview/index.tsx

```diff
@@ -531,3 +531,77 @@ ReactDOM.render(
     </div>,
     document.getElementById("table-9"),
 );
+
+interface IReorderableTableExampleState {
+    children?: JSX.Element[];
+    data?: any[];
+}
+
+const REORDERABLE_TABLE_DATA = [
+    ["A", "Apple", "Ape", "Albania", "Anchorage"],
+    ["B", "Banana", "Boa", "Brazil", "Boston"],
+    ["C", "Cranberry", "Cougar", "Croatia", "Chicago"],
+    ["D", "Dragonfruit", "Deer", "Denmark", "Denver"],
+    ["E", "Eggplant", "Elk", "Eritrea", "El Paso"],
+].map((array) => ({
+    // tslint:disable:object-literal-sort-keys
+    letter: array[0],
+    fruit: array[1],
+    animal: array[2],
+    country: array[3],
+    city: array[4],
+    // tslint:enable:object-literal-sort-keys
+}));
+
+class ReorderableTableExample extends React.Component<{}, IReorderableTableExampleState> {
+    public state = {
+        data: REORDERABLE_TABLE_DATA,
+    } as IReorderableTableExampleState;
```

Comment:
always better to declare type up front instead of casting. sometimes you lose type safety when casting, which you don't lose if you type the var first.

`public state: IReorderableTableExampleState = { ... }`